server/modules/authentication/discord/authentication.js

```diff
@@ -6,6 +6,18 @@
 
 const DiscordStrategy = require('passport-discord').Strategy
 const _ = require('lodash')
+const DiscordOauth2 = require('./node_modules/discord-oauth2/index.js') //I don't remember why I 
+
+// Checks for the existence of all of the required role IDs in the member's guild role IDs.
+function hasRoles(memberRoles, authRoles) {
+  if (authRoles.every(value => {
+    return memberRoles.includes(value)
+  })) {
+    return true
+  } else {
+    return false
+  }
+};
```

Comment:
Just a small nit: this function can be transformed into one line - I would suggest removing this function and move the logic of if below.